app/controllers/polls_controller.rb

```diff
@@ -0,0 +1,14 @@
+class PollsController < ApplicationController
+  #user is at picture page, route meals#setup
+  def create
+    @poll = Poll.new(params.require(:poll).permit(:score, :meal, :cuisine))
```

Comment:
poll instance carries a cuisine id instead of cuisine instance. Need to find cuisine by id first then assign this cuisine into poll.